keps/sig-auth/3766-referencegrant/README.md

```diff
@@ -441,174 +368,365 @@ rejected".
 ```
 
-### Outstanding questions and clarifications
+The new version communicates the scope more clearly because `group`+`resource`
+is unambiguous and corresponds to exactly one set of objects on the API Server.
 
-This section lists some of the outstanding questions people have had about
-ReferenceGrant, and other items that we'll be clarifying in the godoc and other
-documentation as part of the transition to the new API group, along with any
-other changes we need to make that aren't already reflected in this document.
+This change also leaves room for an enhancement. Whether we have an in-tree or
+CRD implementation, we can rely on the exact matching that the plural resource
+name gives us, and [warn](https://kubernetes.io/blog/2020/09/03/warnings/) if
+either side of the grant is for an API that's not served by this cluster.
+
+#### 3. Verbs
 
-Also note that we don't consider any of these blockers for the general _idea_ of
-moving ReferenceGrant to the new API group, just notes to save discussion time.
+A new "verbs" field will be added in response to [earlier
+feedback](https://github.com/kubernetes/enhancements/pull/3767#discussion_r1084509958)
+on this KEP. This allows users to clearly communicate the intent of the
+ReferenceGrant. This will be particularly useful if there's ever a kind of
+resource that can make more than one kind of cross-namespace reference, or a
+situation where the same resource could receive cross-namespace references for
+different purposes.
 
-* Clarify that an implementation is required to reconcile ReferenceGrant for
-  specific `To` Kinds.
-* Corollary for future work, define how controllers interact. Is it a problem if
-  multiple controllers consume the same ReferenceGrant?
-* Status design is still pending, but it's currently expected that controllers
-  will indicate status on the _referring_ resources, not on ReferenceGrant itself.
-<<[UNRESOLVED do we need status? ]>>
-Original Thread: https://github.com/kubernetes/enhancements/pull/3767#discussion_r1084670421
+We will initially define 3 well-known values here to represent the current use
+cases of ReferenceGrant, and we will leave room for other custom domain-prefixed
+values if this API is used for implementation-specific use cases in the future.
 
-We want to be able to represent the following, in descending order of 
-importance: 
+Future standard verbs will be approved by SIG Auth. In general, standard verbs
+should be reserved for:
 
-1. Communicate that the ReferenceGrant is actively being used
-2. Communicate which controllers are using this ReferenceGrant
-3. Communicate how many times it's been used with sufficient granularity that I
-   can see the effects of my changes (if I remove this reference, am I removing
-   a dependency on this ReferenceGrant?)
+* References to and from official Kubernetes APIs
+* Well defined use cases that will have conformance tests
+* Concepts that will be broadly available in Kubernetes clusters
 
```

Comment:
- If I try to create a grant using a verb that isn't allowed, what happens?
- If I create a grant using a verb that is allowed, then downgrade to a Kubernetes release that doesn't allow the verb, what happens?

(for alpha, we can IMO mark that as an unresolved question and nonetheless move forward, with the `verbs` field behind its own feature gate)